blocks/v2-resources-gallery/v2-resources-gallery.js

```diff
@@ -0,0 +1,113 @@
+import { wrapImageWithVideoLink, selectVideoLink, isVideoLink } from '../../scripts/video-helper.js';
+import { createElement } from '../../scripts/common.js';
+
+const blockName = 'v2-resources-gallery';
+
+export default function decorate(block) {
+  const blockHeading = block.querySelector('div:first-child');
+  blockHeading.classList.add(`${blockName}__heading`);
+
+  const viewAllButton = createElement('button', {
+    classes: ['v2-resources-gallery__toggle-resources', 'tertiary'],
```

Comment:
use block name here `${blockName}__toggle-resources`. Same in line 18, 22

Reply:
Done